src/js/jwt/insights/user.js

```diff
@@ -63,7 +66,10 @@ module.exports = (token) => {
                 }
             }
 
-            return user;
+            return {
```

Comment:
@karelhala @ryelo 
I am pretty sure we want:

```javascript
  user.identity.entitlements = data;
  return user;
```
Ryan do you know what the 3scale object is going to look like... I think it is:

```
{
  "identity": {
    "account_number": "6089719",
    "type": "User",
    "user": {
      "username": "rhn-insightsqe-ihands",
      "email": "ihands+qa@redhat.com",
      "first_name": "Ian",
      "last_name": "Hands",
      "is_active": true,
      "is_org_admin": true,
      "is_internal": false
    },
    "internal": {
      "org_id": 52399467
    },
    "entitlements": {
      "hybrid_cloud": {
        "is_entitled": true
      },
      "insights": {
        "is_entitled": true
      },
      "openshift": {
        "is_entitled": true
      },
      "smart_management": {
        "is_entitled": false
      }
    }
  }
}
```